packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue

```diff
@@ -0,0 +1,395 @@
+<template>
+
+  <ImmersivePage
+    :appBarTitle="$tr('toolbarHeader')"
+    :route="backRoute"
+    :icon="icon"
+  >
+    <KPageContainer v-if="device" :style="pageHeight">
+      <KGrid gutter="48" class="edit-sync-schedule">
+
+        <KGridItem>
+          <h1>{{ $tr('editSyncScheduleSubTitle') }}</h1>
+        </KGridItem>
+
+        <KGridItem>
+          <p>{{ device.device_name }} </p>
+        </KGridItem>
+
+        <KGridItem class="select-element">
+          <div
+            class="k-select"
+          >
+            <KSelect
+              v-model="selectedInterval"
+              class="selector"
+              :style="selectorStyle"
+              :options="selectArray"
+              label="Repeat"
+            />
+
+          </div>
+          <div
+            v-if="selectedItem.label !== 'Never'
```

Comment:
You should do this based on `selectedItem.value` here, otherwise this will only work when the interface is in English.

Fortunately that makes the comparison a bit more concise:

```
v-if="selectedItem.value <= 86400"
```

You just need to check if it's equal to the value for every day or less!